go.mod

```diff
@@ -174,6 +174,11 @@ require (
 	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 )
 
+// Do not remove. It breaks github.com/gravitational/go-oidc.
+require (
+	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
```

Comment:
I don't need to do anything special for `go mod tidy` to pick this up. I wonder why it seems to consistently get dropped for you.